modules/ROOT/pages/_partials/mule-runtime-metrics.adoc

```diff
@@ -1,66 +1,79 @@
-= Usage and Pricing Metrics
+// tag::intro[]
+Mule runtime reports track your use of Mule messages, Mule flows, and data throughput. 
 
-MuleSoft captures usage data for Mule flows, Mule messages, and data throughput, but not all of this data is aggregated in usage reports. 
+To track Mule message usage, the runtime report counts the number of times a Mule event source triggers a Mule message. You can view the number of these messages in a given day or month by business group, environment, and application. 
 
-For select customers in the US control plane, MuleSoft offers a pricing and packaging model for Anypoint Platform that allots a number of Mule flows, Mule messages, and data throughput (measured as network I/O bytes). Not all types of Mule flows and messages count toward the allotments in a package. 
+To calculate usage, MuleSoft meters and aggregates the number of messages on a daily and monthly basis. After a message is triggered, the report does not track changes to the message because the message is processed within the application’s flows.
```

Comment:
```suggestion
To calculate usage, MuleSoft meters and aggregates the number of messages on a daily and monthly basis. After a message is triggered, the report doesn’t track changes to the message because the message is processed within the application’s flows.
```

Reply:
For "daily and monthly basis", can you just say "daily and monthly"?